shared_model/cryptography/crypto_provider/crypto_model_signer.hpp

```diff
@@ -6,19 +6,17 @@
 #ifndef IROHA_CRYPTO_MODEL_SIGNER_HPP_
 #define IROHA_CRYPTO_MODEL_SIGNER_HPP_
 
+#include "cryptography/crypto_provider/abstract_crypto_model_signer.hpp"
 #include "cryptography/crypto_provider/crypto_signer.hpp"
 
-namespace shared_model {
+#include "interfaces/iroha_internal/block.hpp"
```

Comment:
it is a very minor issue, but again, why the includes are divided into two groups?

Reply:
`CryptoModelSigner` inherits from `AbstractCryptoModelSigner` and uses `CryptoSigner` as default template.